app/scripts/modules/core/pipeline/config/triggers/git/gitTrigger.html

```diff
@@ -0,0 +1,46 @@
+<ng-form name="cronForm">
+  <div class="form-group">
+    <label class="col-md-3 sm-label-right">
+      Repository Type
+      <help-field key="pipeline.config.git.repotype"></help-field>
+    </label>
+    <div class="col-md-6">
+      <select
+        class="form-control input-sm"
+        ng-options="repoType for repoType in ['stash', 'github']"
+        ng-model="vm.trigger.source">
+        <option value="">Select Repo Type</option>
+      </select>
+    </div>
+  </div>
+  <div class="form-group">
+    <label class="col-md-3 sm-label-right">
+      Project
+      <help-field key="pipeline.config.git.project"></help-field>
+    </label>
+
+    <div class="col-md-9">
+      <input type="text" class="form-control input-sm"
+             ng-model="vm.trigger.project"
+             name="project"
+             required
+             placeholder="Enter your repository project name, i.e. spinnaker for github.com/spinnaker/echo"
```

Comment:
Inline examples are helpful but are we sure we've got the terminology right?

`spinnaker` is the owner ... not really a project at least in the case of github which generally labels repositories themselves as `projects` AFAIK. (On the contrary, Stash does treat groups of repositories as a project).

Similarly below it might be more common to ask for the repository slug name vs "source repository name".

Not a huge deal either way.